lib/collection/src/shards/shard_holder/resharding.rs

```diff
@@ -391,6 +497,39 @@ impl ShardHolder {
     }
 }
 
+#[derive(Clone, Debug, Default)]
+pub struct OperationsByMode {
+    pub update_all: Vec<CollectionUpdateOperations>,
+    pub update_existing: Vec<CollectionUpdateOperations>,
+}
+
+impl OperationsByMode {
+    pub fn update_existing(mut self, operation: CollectionUpdateOperations) -> Self {
+        match operation {
+            CollectionUpdateOperations::PointOperation(
+                point_ops::PointOperations::UpsertPoints(operation),
+            ) => {
+                self.update_existing = operation.into_update_only();
+            }
```

Comment:
Convert `upsert` operation into "update vector"/"overwrite payload", if we want to apply `upsert` in "update existing" mode.

Reply:
Is this still relevant?

Reply:
It is. We don't have any other way to handle this rn.

Reply:
I believe it is handled in `into_update_only()`